### PR TITLE
add support for installing QEMU

### PR DESCRIPTION
**Pull request by gdams**
Adds support for Qemu, in combination with docker this can be very useful for building/testing multiarch images

### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by timja — COMMENTED** [review]
Thanks, looks good, what testing have you done?

File: src/main/webapp/help-initScript.html [hunk 1]
```diff
@@ -77,3 +77,8 @@
     </ol>
   </o>
 </div>
+
+# Install QEMU
+sudo apt install -y qemu-user-static
+
+qemu --version
```
[the remark this answers is not in the document]
Comment (gdams) [reply src/main/webapp/help-initScript.html:84]: I wasn't sure what this file was TBH, happy to remove it.

**Comment by gdams** [comment]
> Thanks, looks good, what testing have you done?

not much, I wasn't sure if the current CI tests would pick this up?

File: src/main/webapp/help-initScript.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (timja) [reply src/main/webapp/help-initScript.html:84]: inline documentation

**Comment by timja** [comment]
> > Thanks, looks good, what testing have you done?
> 
> not much, I wasn't sure if the current CI tests would pick this up?

no it won't as CI doesn't spin up Azure infra, there is some integration tests that can be manually run but I generally test most changes to this plugin manually when it interacts with Azure 😢

File: src/main/webapp/help-initScript.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (gdams) [reply src/main/webapp/help-initScript.html:84]: reverted this

**Comment by gdams** [comment]
> > > Thanks, looks good, what testing have you done?
> > 
> > 
> > not much, I wasn't sure if the current CI tests would pick this up?
> 
> no it won't as CI doesn't spin up Azure infra, there is some integration tests that can be manually run but I generally test most changes to this plugin manually when it interacts with Azure 😢

Let me try and spin up a test CI environment

**Comment by gdams** [comment]
@timja I've confirmed this this works on Ubuntu 22.04 and 20.04. I'll do some windows testing in my environment next

Screenshot 2024-10-09 at 12 59 46

**Comment by gdams** [comment]
@timja what's the best way to test this? I've redeployed a new plugin and now for some reason Jenkins won't detect it as a cloud plugin so won't let me create a new cloud. I feel like there may be a better way?

**Comment by timja** [comment]
> @timja what's the best way to test this? I've redeployed a new plugin and now for some reason Jenkins won't detect it as a cloud plugin so won't let me create a new cloud. I feel like there may be a better way?

Running it locally with `mvn hpi:run` will get you a test instance of Jenkins running on port 8080, thats how I test it

**Comment by gdams** [comment]
> > @timja what's the best way to test this? I've redeployed a new plugin and now for some reason Jenkins won't detect it as a cloud plugin so won't let me create a new cloud. I feel like there may be a better way?
> 
> Running it locally with `mvn hpi:run` will get you a test instance of Jenkins running on port 8080, thats how I test it

I managed to get this running on Windows too so I think this is good to go

**Review by timja — APPROVED** [review]
Thanks!

**Pull request merged**